util.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+main() {
+  case $1 in
+  "")
+    echo "Kindly supply a command" && exit 0
+    ;;
+  move)
```

Comment:
another capability I can see being useful here would be copying the source dir from the vm to the local machine. I will often lazily use `vi` try out a change on the vm to see if it works.

`move` doesn't make it obvious which direction the files are going so especially if we had the above option, we'll want to come up with something a little less ambiguous (`deploy`?)